Integrations/ARIA/aria_packet_intelligence.yml

```diff
@@ -0,0 +1,1625 @@
+      description: 'Returns endpoints information, such as the IP address, about the
+        SIAs that were modified based on the rule change.'
+      type: string
+    description: Creates a rule that drops all packets matching the specified 5-tuple
+      values.
+  - name: aria-unblock-conversation
+    arguments:
+    - name: rule_name
+      required: true
+      description: The name of the rule to delete.
+    - name: label_sia_group
+      description: 'The name of the group to which the SIA belongs. NOTE: Only two
+        labels are allowed. If you enter values for all three labels, the command
+        will fail. Also, if no labels are provided (i.e., they are empty), the rule
+        is deleted to every SIA attached to the SDSo.'
+    - name: label_sia_name
+      description: 'The name of the SIA. NOTE: Only two labels are allowed. If you
+        enter values for all three labels, the command will fail. Also, if no labels
+        are provided (i.e., they are empty), the rule is deleted to every SIA attached
+        to the SDSo.'
+    - name: label_sia_region
+      description: 'The name of the region to which the SIA belongs. NOTE: Only two
+        labels are allowed. If you enter values for all three labels, the command
+        will fail. Also, if no labels are provided (i.e., they are empty), the rule
+        is deleted to every SIA attached to the SDSo.'
+    outputs:
+    - contextPath: Aria.UnblockConversation.Rule
+      description: 'Specifies the name of the rule and the settings that define the rule.'
+      type: string
+    - contextPath: Aria.UnblockConversation.Status
+      description: 'Returns the response code, the state of the command, and the
+        timestamp indicating when the command completed. If an error occurs or the
+        response code is not 201, this also returns information about the error.'
+      type: string
+    - contextPath: Aria.UnblockConversation.Endpoints
+      description: 'Returns endpoints information, such as the IP address, about the
+        SIAs that were modified based on the rule change.'
+      type: string
+    description: Deletes a named rule from the 5-tuple logic block. This allows the
+      previously blocked conversation to resume.
+  - name: aria-record-conversation
+    arguments:
+    - name: src_ip
+      required: true
+      description: The source IP address.
+    - name: src_port
+      description: The source port(s). This accepts a comma-separated list (e.g.,
+        “1, 3”), a range (e.g., “1-3”), or a combination (e.g., “1, 3-5”).
+    - name: target_ip
+      required: true
+      description: The destination IP address.
+    - name: target_port
+      description: The destination port(s). This accepts a comma-separated list (e.g.,
+        “1, 3”), a range (e.g., “1-3”), or a combination (e.g., “1, 3-5”).
+    - name: protocol
+      description: The protocol (e.g., TCP) used for the packets.
+    - name: vlan_id
```

Comment:
will the user know how to retrieve this ID ?

Reply:
Users knows that. The VLAN ID your network switch uses to forward packets to the Packet Recorder.

Reply:
👍